### ISSUE TITLE
Teamspeak not resized properly

### ISSUE THREAD

**Issue by Toriniasty** [digitalcreations/MaxTo#481]
**Describe the bug**
Trying to maximise Teamspeak according to my region. Unfortunately MaxTo doesn't maximise according to the region, it just goes fullscreen.

**To Reproduce**
1. Install TeamSpeak 3.3.2
2. Set MaxTo regions
3. Start TeamSpeak
4. Maximise

**Expected behavior**
TeamSpeak is sized accordingly to MaxTo region.

**System information:**
 - Windows version: 10, 1903
 - MaxTo version 2.0.1

**Comment by vegardlarsen** [comment]
- Does it work with other programs, e.g. simple ones like Notepad?
- Does it work after a reboot?
- Do the keyboard shortcuts work for TeamSpeak?

**Comment by Toriniasty** [comment]
Does it work with other programs, e.g. simple ones like Notepad?  
Yes. Skype/Whatsapp/Spotify are fine.  
Does it work after a reboot?  
No change after reboot  
Do the keyboard shortcuts work for TeamSpeak?
If you mean internal TeamSpeak shortcuts I think so, MaxTo not sure. Will double check in the evening.

**Comment by vegardlarsen** [comment]
I'll be trying to verify this myself, and see what the issue can be, to see if we can work around it.

**Comment by Toriniasty** [comment]
Just tried MaxTo shortcuts in Teamspeak, they do not work.   
Native Windows shortcuts do work, tested Win+Up/Left/Right.

**Comment by vegardlarsen** [comment]
@Toriniasty Something does not compute here; do MaxTo's keyboard shortcuts work with other programs? The keyboard shortcuts are entirely separate from the other mechanisms; and therefore should work.

**Comment by Toriniasty** [comment]
Yes, they do. Tried on for example explorer window or total commander. When I tried them on Teamspeak didn't do nothing at all.

**Comment by vegardlarsen** [comment]
I have confirmed that TeamSpeak uses a window style that is incompatible with MaxTo version 2.0.1, and we have already fixed this issue for MaxTo version 2.1.0. This issue will be fixed in the next preview release of MaxTo.
